lib/datalog/roletester/proto/access.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+package types;
+
+// Facts represents a datalog database of predicates
+message Facts {
+    enum PredicateType {
+        HASROLE = 0;
+        HASTRAIT = 1;
+        ROLEALLOWSLOGIN = 2;
+        ROLEDENIESLOGIN = 3;
+        ROLEALLOWSNODELABEL = 4;
+        ROLEDENIESNODELABEL = 5;
+        NODEHASLABEL = 6;
+        // Outputs
+        HASACCESS = 7;
+        DENYACCESS = 8;
+        DENYLOGINS = 9;
```

Comment:
Nit: All caps is not really Go style, let's rename to `HasRole`, `HasTrait`, `RoleAllowsLogin`, etc.